CHANGELOG.md

```diff
@@ -11,7 +11,7 @@ We use *breaking* word for marking changes that are not backward compatible (rel
 
 ## Unreleased
 
-## [v0.11.0-rc.0](https://github.com/thanos-io/thanos/releases/tag/v0.11.0-rc.0) - 2020.02.19
+## [v0.11.0-rc.1](https://github.com/thanos-io/thanos/releases/tag/v0.11.0-rc.1) - 2020.02.26
 
 ### Fixed
 
```

Comment:
Fix that we have line below. Was this introduced by v0.10.1? If not then we should drop this line, I think.

Reply:
Yeah. I was debating the same here with Frederic. We want to keep it for the RC, but drop it for the actual release, I guess?!